internal/langserver/handlers/command/module_calls_test.go

```diff
@@ -18,7 +18,7 @@ func Test_parseModuleRecords(t *testing.T) {
 			records: []datadir.ModuleRecord{
 				{
 					Key:        "ec2_instances",
-					SourceAddr: "terraform-aws-modules/ec2-instance/aws",
+					SourceAddr: "registry.terraform.io/terraform-aws-modules/ec2-instance/aws",
```

Comment:
Would you mind adding this as a separate test case, e.g. `Test_parseModuleRecords_v1_1` ? Just to make sure we keep testing for both old and new format.

Reply:
We have an entry in our test table below for `terraform-aws-modules/eks/aws`,but I can create a separate test case as well.